hawc/apps/materialized/models.py

```diff
@@ -0,0 +1,59 @@
+from django.apps import apps
+from django.core.cache import cache
+from django.db import connection, models, transaction
+
+from . import managers
+
+
+class MaterializedView(models.Model):
+    class Meta:
+        abstract = True
+        managed = False
+
+    @classmethod
+    def set_refresh_view_cache(cls, bool):
+        cache.set(f"refresh-{cls._meta.db_table}", bool)
+
+    @classmethod
+    def get_refresh_view_cache(cls):
+        cache.get(f"refresh-{cls._meta.db_table}", False)
+
+    @classmethod
+    @transaction.atomic
+    def refresh_view(cls):
+        with connection.cursor() as cursor:
+            cursor.execute(f"REFRESH MATERIALIZED VIEW CONCURRENTLY {cls._meta.db_table}")
+
+
+class Score(MaterializedView):
```

Comment:
`MaterializedScore`? Or is the `materialized` namespace enough? I'm curious what kind of naming convention we should use for these going forward.